src/environments/environment.test.ts

```diff
@@ -1,39 +1,40 @@
 // `.env.ts` is generated by the `npm run env` command
-import env from "./.env";
```

Comment:
revert change from " to '

Reply:
@amydoxym i guess we maintain ', as the other environment files are using same.